src/status_im/common/floating_button_page/floating_container/style.cljs

```diff
@@ -1,9 +1,8 @@
-(ns status-im.common.floating-button-page.floating-container.style
-  (:require [react-native.safe-area :as safe-area]))
+(ns status-im.common.floating-button-page.floating-container.style)
 
 (defn content-container
-  [blur? keyboard-shown?]
-  (let [margin-bottom (if keyboard-shown? 0 (safe-area/get-bottom))]
+  [blur? keyboard-shown? bottom-inset]
```

Comment:
@OmarBasem I think you mentioned this in another pr about this floating button page - the safe insets here are breaking the bottom margin on some pages - I made this value configurable for this reason. perhaps there's a better approach. Additionally I can move this outside this pr 👍 
cc @ulisesmac

Reply:
No problem we can look into it in a separate issue

Reply:
ok, I'll remove from my changes from here to not make the mess bigger :)